beacon-chain/rpc/prysm/v1alpha1/validator/proposer_bellatrix.go

```diff
@@ -270,23 +281,17 @@ func (vs *Server) getPayloadHeaderFromBuilder(
 		return nil, errors.Wrap(err, "could not validate builder signature")
 	}
 
-	maxBlobsPerBlock := params.BeaconConfig().MaxBlobsPerBlock(slot)
-	var kzgCommitments [][]byte
-	if bid.Version() >= version.Deneb {
-		kzgCommitments, err = bid.BlobKzgCommitments()
-		if err != nil {
-			return nil, errors.Wrap(err, "could not get blob kzg commitments")
+	var executionRequests *enginev1.ExecutionRequests
+	if bid.Version() >= version.Electra {
+		eBid, ok := bid.(builder.BidElectra)
+		if !ok {
+			return nil, errors.New("builder returned non-electra bid")
```

Comment:
Related to my comment above about `if ok` - here you do return an error, so let's pick one approach and be consistent about this